Maria-Edgeworth-Letters-Project/me-tei
# License cc

*** I'm including everyone on this PR so that all are aware of changes, but I only need one approval. I'll merge myself if approved. DO comment if you disagree with the license value. ***

## What does this Pull Request do?

At our October project meeting we discussed what license would be the most appropriate to use for the TEI and other project deliverables. The following resources were shared to justify the use of CC BY:
- https://www.tapasproject.org/tei-learning-docs-bc/files/leary-placeography
- https://mapoflondon.uvic.ca/legal.htm 
- Official TEI C Guidelines - https://tei-c.org/guidelines/licensing-and-citation/ - we should use CC+BY for TEI
- Article - https://journals.openedition.org/jtei/2610 
- [Free Knowledge Thanks to Creative Commons Licenses: Why a Non-commercial Clause Often Won’t Serve Your Needs.](https://commons.wikimedia.org/wiki/File:Free_Knowledge_thanks_to_Creative_Commons_Licenses.pdf)
In particular, the following language on the TEI Consortium Guidelines page was influential: "The CC+BY licence is generally appropriate for usages which treat TEI content as data or documentation." The guidelines do link to 3.0 of the license, but I believe using the most recent license (4.0) is best.

Given this, this PR updates all of the license values across the encoded letters, the TEI template, and the "-ography" files. Note that updating a license is problematic because once assigned a CC license cannot be revoked. Previous versions of our TEI are available on GitHub since this repository is public. I'm going to argue that the previous license was not valid because we did not include a URL in the target attribute. According to [Hannesschläger](https://journals.openedition.org/jtei/2610): "The full license text must be either included directly in the document (which might enlarge TEI files in a way you are not happy with), or properly referenced by means of linking. Merely mentioning the license, however well-known it may be, is not sufficient. Therefore, including the link to the license in the @target attribute of the <licence> element is essential when applying a license to a TEI file." I'm stressing this here because it would be highly inappropriate for us to make changes to the license value in the future. Comment if you disagree with the value selected.

## How should this be tested?

- Do you agree with the decision to use CC BY for our TEI? If not, comment and explain your reasoning.
- Do any changes affect the validity of the XML?
- Is the way in which the license is communicated (with a target URL attribute and the text "Distributed under a Creative Commons Attribution 4.0 (CC BY 4.0) Unported License") the best way to represent this information?
- Have any files not been addressed that should be for this update?

## Additional Notes

If this is merged, everyone should be sure to update the version of the template they are using.
